pkg/controllers/interruption/metrics.go

```diff
@@ -35,15 +35,15 @@ var (
 			Help:      "Count of messages received from the SQS queue. Broken down by message type and whether the message was actionable.",
 		},
 		[]string{messageTypeLabel},
-	)
+	) //stability=STABLE
```

Comment:
Also, can we put these on newlines above the definition of the metric like you typically would for a comment?